src/services/opsworksApp/format.ts

```diff
@@ -0,0 +1,89 @@
+import { generateUniqueId } from '@cloudgraph/sdk'
+
+import { RawAwsOpsWorksApp } from './data'
+import { AwsOpsWorksApp } from '../../types/generated'
+import { opsworksAppArn } from '../../utils/generateArns'
+
+export default ({
+  service,
+  account: accountId,
+  region,
+}: {
+  service: RawAwsOpsWorksApp
+  account: string
+  region: string
+}): AwsOpsWorksApp => {
+  const {
+    AppId: appId,
+    StackId: stackId,
+    Shortname: shortname,
+    Name: name,
+    Description: description,
+    DataSources: dataSources,
+    Type: type,
+    AppSource: appSource,
+    Domains: domains,
+    EnableSsl: enableSsl,
+    SslConfiguration: sslConfiguration,
+    Attributes: attributes,
+    CreatedAt: createdAt,
+    Environment: environment,
+  } = service
+
+  const arn = opsworksAppArn({ region, account: accountId, name: name })
```

Comment:
@tyler-dunkel @m-pizarro opsworksApp doesn't have ARN field and I can't find how the opsworksApp ARN looks like. So I just make the ARN in this format. We may need to update this.
`arn:aws:opsworks:${region}:${account}:app/${name}`

Reply:
I see opsworks apps have an `appId` (thats what is used to query it with the api). It might be /${app} not /${name} unless name === appId